examples/hydra-vue-f7/src/components/richtext.vue

```diff
@@ -1,22 +1,33 @@
 <template>
-    <f7-link v-if="node.type =='link'" :href="node.data.url" external :data-node-id="node['nodeId']">{{ node.text }}<RichText v-for="child in subs" :key="child" :node="child"/></f7-link>
-    <template v-else-if="!node.type">{{ node.text }}</template>
-    <component v-else :is="node.type" :data-node-id="node['nodeId']">{{ node.text }}<RichText v-for="child in subs" :key="child" :node="child"/></component>
+  <template v-if="node.type === 'link'">
+    <f7-link :href="node.data.url" external :data-node-id="node.nodeId">
+      {{ node.text }}
+      <RichText v-for="child in subs" :key="child.nodeId" :node="child" />
+    </f7-link>
+  </template>
+  <span v-else-if="!node.type" :data-node-id="node.nodeId">
```

Comment:
I do think it's not needed to force wrapping text in a span and give it a node-id. I think there is enough information conveyed to hydra to work out where the cursor is without having to force more ugly markup.
But this can be looked at later. it's fine for now.